slidetoact/src/main/java/com/ncorti/slidetoact/SlideToActView.kt

```diff
@@ -204,9 +204,28 @@ class SlideToActView @JvmOverloads constructor(
     private lateinit var mDrawableArrow: VectorDrawableCompat
 
     /** Tick drawable, is actually an AnimatedVectorDrawable */
-    private val mDrawableTick: Drawable
+    private var mDrawableTick: Drawable
     private var mFlagDrawTick: Boolean = false
 
+    var drawableTick: Int = 0
```

Comment:
```suggestion
    var completeIcon: Int = 0
```

To be consistent with `sliderIcon`. Tick is an internal name and should not be exposed

Reply:
done